R/help_create_blogpost.R

```diff
@@ -0,0 +1,151 @@
+library(stringr)
+library(tidyverse)
+library(checkmate)
+library(rlang)
+
+#' function to create a new post skeleton
+#'
+#' @param post_name string
+#' @param author string (can be vector for more authors)
+#' @param post_date atomic character date in the format "%Y-%m-%d", e.g. "2023-06-15", defaults to today
+#' @param description blog post description, can be left empty and filled later
+#' @param image one of a set of options for default images
+#' @param tags character vector, some of a set of options
+#'
+#' @return just a message when it worked. Side effect: creates skeleton of blogpost
+#' @export
+#'
+#' @examples
+#' create_post(post_name = "this is my new example post", author = "Stefan Thoma", cover_image = "admiral", tags = "admiral")
+#'
+create_post <- function(post_name,
+                        author = Sys.info()["user"],
+                        post_date = format(Sys.time(), "%Y-%m-%d"),
+                        description = "",
+                        cover_image = available_images,
+                        tags = c(
+                          "metadata",
+                          "submission",
+                          "qc",
+                          "ADaMs",
+                          "SDTMs",
+                          "community",
+                          "conferences",
+                          "admiral",
+                          "roak",
+                          "xportr",
+                          "metatools",
+                          "metacore"
+                        )) {
+  path_to_img <- "media"
+  available_images <- list.files(path_to_img) %>% tools::file_path_sans_ext()
```

Comment:
This line is crashing for me 
![image](https://github.com/pharmaverse/blog/assets/10111024/04eb6efa-a8b0-49a4-a93d-ad52cbdfba7f)
is it working on your end?

Reply:
The line that crashed was actually the rlang::arg_match line for me. 
Will update and then try out on posit cloud.